### PR TITLE
Adding Kafka under third party tests

### PR DESCRIPTION
**Pull request by suman-mitra**
This PR is created to add Kafka under third party applications test. Below is the personal build which ran with the changes under my fork.

https://ci.adoptopenjdk.net/view/work%20in%20progress/job/test_personal/629/consoleFull

Build was successful but tests failed, for which we will raise defects..
Please review the changes and approve

### CODE REVIEW

File: thirdparty_containers/kafka/dockerfile/kafka-test.sh [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+type -p java
+echo "java -version is: \n"
+java -version
+
+# Replace the following with the initial command lines that trigger execution of your test
+cd /kafka
+ls .
+pwd
+
+
```
Comment (Mesbah-Alam) [inline thirdparty_containers/kafka/dockerfile/kafka-test.sh]: Please remove extra space

File: thirdparty_containers/kafka/dockerfile/kafka-test.sh [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+echo "Building kafka  using gradle" && \
+gradle -q
+./gradlew jar
+
+
```
Comment (Mesbah-Alam) [inline thirdparty_containers/kafka/dockerfile/kafka-test.sh]: Please remove extra spaces

File: thirdparty_containers/kafka/playlist.xml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+-->
+<playlist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../TestConfig/playlist.xsd">
+	<test>
+		<testCaseName>kafka_test</testCaseName>
+	<command>docker run --rm -v $(JDK_HOME):/java  adoptopenjdk-kafka-test:latest "jvm pos neg"; \
```
Comment (TianyuZuo) [inline thirdparty_containers/kafka/playlist.xml]: Are these `"jvm pos neg"` in the command section also used for Kafka tests?

File: thirdparty_containers/kafka/dockerfile/kafka-test.sh (hunk 1 above)
Comment (suman-mitra) [reply thirdparty_containers/kafka/dockerfile/kafka-test.sh]: removed

File: thirdparty_containers/kafka/dockerfile/kafka-test.sh (hunk 2 above)
Comment (suman-mitra) [reply thirdparty_containers/kafka/dockerfile/kafka-test.sh]: removed

File: thirdparty_containers/kafka/dockerfile/kafka-test.sh [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+TEST_SUITE=$1
+
+echo "PATH is : $PATH"
+echo "JAVA_HOME is : $JAVA_HOME"
+echo "type -p java is :"
+type -p java
+echo "java -version is: \n"
+java -version
+
+# Replace the following with the initial command lines that trigger execution of your test
```
[the remark this answers is not in the document]
Comment (suman-mitra) [reply thirdparty_containers/kafka/dockerfile/kafka-test.sh]: done..Replaced with "Initial command to trigger the execution of kafka test"

File: thirdparty_containers/kafka/dockerfile/Dockerfile [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+	apt-transport-https \
+	ca-certificates \
+	dirmngr \
+	curl \
+	git \
+	make \
+	unzip \
+	vim \
+	wget
+
```
[the remark this answers is not in the document]
Comment (suman-mitra) [reply thirdparty_containers/kafka/dockerfile/Dockerfile]: removed

File: thirdparty_containers/kafka/dockerfile/Dockerfile [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+#Install Gradle
+RUN mkdir -p /usr/share/gradle \
+         && cd /usr/share/gradle \
+         && wget -q https://services.gradle.org/distributions/gradle-4.5-bin.zip \
+         && unzip -qq gradle-4.5-bin.zip \
+         && rm -rf gradle-4.5-bin.zip \
+         && ln -s /usr/share/gradle/gradle-4.5/bin/gradle /usr/bin/gradle \
+         && cd /
+
```
[the remark this answers is not in the document]
Comment (suman-mitra) [reply thirdparty_containers/kafka/dockerfile/Dockerfile]: removed

File: thirdparty_containers/kafka/playlist.xml (hunk 3 above)
Comment (suman-mitra) [reply thirdparty_containers/kafka/playlist.xml]: Not needed, have removed them now

**Pull request merged**